webui/src/components/ReplicasetList/ReplicasetList.js

```diff
@@ -116,7 +116,25 @@ const styles = {
   `
 };
 
-const prepareReplicasetList = dataSource => [...dataSource].sort((a, b) => b.uuid < a.uuid ? 1 : -1);
+const prepareReplicasetList = dataSource => [...dataSource].sort((a, b) => {
+  let aValue = a.alias || '';
+  let bValue = b.alias || '';
+
+  if (aValue === bValue) {
+    aValue = a.servers[0].alias || '';
+    bValue = b.servers[0].alias || '';
+  }
+
+  if (aValue === bValue) {
+    aValue = a.uuid;
+    bValue = b.uuid;
+  }
+
+  if (aValue === '' || bValue === '') {
```

Comment:
what the case is it for?

Reply:
Replicasets with empty aliases (or empty server aliases) will be at the end.